lib/js/jsre.nim

```diff
@@ -4,12 +4,13 @@
 runnableExamples:
   let jsregex: RegExp = newRegExp(r"\s+", r"i")
   jsregex.compile(r"\w+", r"i")
-  doAssert jsregex.test(r"nim javascript")
-  doAssert jsregex.exec(r"nim javascript") == @["nim".cstring]
-  doAssert jsregex.toString() == r"/\w+/i"
+  assert jsregex.test(r"nim javascript")
```

Comment:
how about introduce `match` and deprecate `test` as a deprecated alias so that it fits better with nim's various regex modules (re, nre, pkg/regex) ?
(sorry i didn't spot this in https://github.com/nim-lang/Nim/pull/14870)
should be done in a followup PR though to avoid mixing cleanups and API changes